config/v1/types_feature.go

```diff
@@ -35,6 +35,9 @@ var (
 	// Because of its nature, this setting cannot be validated.  If you have any typos or accidentally apply invalid combinations
 	// your cluster may fail in an unrecoverable way.
 	CustomNoUpgrade FeatureSet = "CustomNoUpgrade"
+
+	// TopologyManager enables ToplogyManager support. Upgrades are enabled with this feature.
+	TopologyManager FeatureSet = "TopologyManager"
```

Comment:
@derekwaynecarr this name looks odd.  Is this related to high maintenance, high performance clusters?

Reply:
I talked to @sjenning about this and he mentioned that if it's called 'High Performance', then everyone might see that and want to turn it on.

Reply:
There is some talk about High Performance using a RT kernel as well, so whatever name we come up with should describe the feature.

Reply:
alternative names:

- performance sensitive
- latency sensitive

@deads2k do you see an issue if in future we let you transition from "LatencySensitive" to "Default" set?

Reply:
> @deads2k do you see an issue if in future we let you transition from "LatencySensitive" to "Default" set?

I think I'm more inclined to change the meaning of default to match "LatencySensitive". 

LatencySensitive works. PerformanceSensitive will be one that everyone tries to turn on, so I would avoid it.